plus2e/plus2e_tv.mk

```diff
@@ -0,0 +1,24 @@
+# Copyright (C) 2020 The Android Open Source Project
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+$(call inherit-product, device/glodroid/plus2e/device.mk)
+
+PRODUCT_BOARD_PLATFORM ?= H3_Quad-core_Cortex-A7
+PRODUCT_NAME := plus2e_tv
+PRODUCT_DEVICE := plus2e
```

Comment:
PRODUCT_NAME and PRODUCT_DEVICE should have same values, otherwise could be run-time issues.

Reply:
Done.